### DISCUSSION TITLE
Overlaying a Blur on the Map

### DISCUSSION THREAD

**Discussion by ShubhamPatilsd** [react-native-maps/react-native-maps#4763] [Q&A]
Hey y'all!

I was browsing Twitter today and found this post: https://twitter.com/soren_iverson/status/1666807361045159936?t=PNGCafROJKvcbgZr_toegw&s=19

It was basically blurring the regions of Google Maps that you haven't explored. I found this to be intriguing and I created a new Expo app and added `react-native-maps` to my project. So far, my research has led me to using heatmaps to mark the points that I've visited. Is there a way to overlay a blur and then disable that blur on a specific coordinate? Not sure if this is an option but wanted to hear possible solutions to this problem.

Thanks!

**Comment by ShubhamPatilsd** [comment]
Perhaps something like this could work?

https://stackoverflow.com/questions/43645172/geojson-multipolygon-with-multiple-holes

**Comment by ShubhamPatilsd** [comment]
ok interesting

```jsx
<Polygon
          coordinates={[
            { latitude: 90, longitude: 180 },
            { latitude: 90, longitude: -180 },
            { latitude: -90, longitude: -180 },
            { latitude: -90, longitude: 180 },
          ]}
          fillColor={"#000"}
        />
```

I'm using this code to fill in the entire earth with a black box. However, there seems to be no fill. If I make the box smaller, however, the fill does appear. Is there a limit on the area that a polygon can have?

**Comment by sgarg15** [comment]
Hey Shubham, were you able to find a suitable solution for the problem? I was looking for something similar.
